src/main/java/seedu/address/model/calendar/Calendar.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.calendar;
+
+import java.util.ArrayList;
+
+/**
+ * Calendar class
+ */
+public class Calendar {
+    private ArrayList<Lesson> lessons;
```

Comment:
Consider using `List<Lesson>` instead of `ArrayList<Lesson>` when declaring the lessons variable. It might give us greater flexibility and allow us to change the implementation later without affecting the rest of the code. It also adheres to best practices by promoting abstraction. We can still instantiate it as an `ArrayList` or another List type as needed.